src/Browser.ts

```diff
@@ -330,21 +332,16 @@ class Browser {
      * Sends a message to a room
      *
      * @param {number} roomId The room ID to send to
-     * @param {string} message The message to send
+     * @param {string} text The message to send
      * @returns {Promise<Message>} A promise that resolves the message that was sent
      * @memberof Browser
      */
-    public async sendMessage(
-        roomId: number,
-        message: string
-    ): Promise<Message> {
-        const res = await this._postKeyed(`chats/${roomId}/messages/new`, {
-            text: message,
+    public async sendMessage(roomId: number, text: string): Promise<Message> {
+        const { id } = await this._postKeyed(`chats/${roomId}/messages/new`, {
+            text,
         });
 
-        const msg = new Message(this._client, res.id, {
-            roomId,
-        });
+        const msg = new Message(this._client, id, { roomId, content: text });
```

Comment:
Why are we initializing content here? If the server changes the message (Parses it for HTML, or other things), that will not be reflected on content anymore apart of message.

Reply:
Good point. It caused friction with the unit test but I guess I can mock the whole message module and thus mock the `_scrapeTranscript` private method to return the `text`. I am also not sure whether it should not be considered a failure state if the server returns a different message than the one sent. Is there a valid use case where we need to know the server changed something? Come to think of it - maybe should throw or have a flag on the message that identifies if the text passed differs from the text received?

Reply:
To be honest, I do not know what to do about that one - if you think that's important, I'll revert that one and mock out the module for the unit test to work.

Reply:
@danbopes - done, finally got time to rework the test to mock out the `Message` module instead and removed the direct passing of the content to the message instance. Let me know if the PR is ready to merge - I'd still like to get your seal of approval first before proceeding.